iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractThreeWindingsTransformerTest.java

```diff
@@ -186,6 +186,94 @@ public void baseTests() {
         assertEquals(count - 1L, network.getThreeWindingsTransformerCount());
     }
 
+    @Test
+    public void invalidSubstationContainer() {
+        thrown.expect(ValidationException.class);
+        thrown.expectMessage("3 windings transformer 'twt': the 3 windings of the transformer shall belong to the substation 'sub'");
```

Comment:
I am going to make a review just from the test review: so you forbid to create a two windings transformer between 2 voltage levels if they are not located in the same substation. Right ? But you allow to create a two windings transformer between 2 voltage levels that are not contained in a substation ? This last functionality could lead to non realistic network for sure.
- If you do that for network with no substation at all: ok but maybe we need a quality check?
- If you ahve another reason, why not going further and allowing two windings transformer between 1 voltage level in a substation and 1 without ?

Reply:
Resolvedn I have miss understood the test.